bessctl/commands.py

```diff
@@ -762,32 +767,37 @@ def add_port(cli, driver, port, args):
         cli.fout.write('  The new port "%s" has been created\n' % ret.name)
 
 
-@cmd('add module MCLASS [NEW_MODULE] [MODULE_ARGS...]', 'Add a new module')
-def add_module(cli, mclass, module, args):
-    cli.bess.pause_all()
+@cmd('add module MCLASS [NEW_MODULE] [PAUSE_WORKERS] [MODULE_ARGS...]',
+     'Add a new module')
+def add_module(cli, mclass, module, pause_workers, args):
+    if pause_workers != 'no_pause':
+        cli.bess.pause_all()
     try:
         ret = cli.bess.create_module(mclass, module, args)
     finally:
-        cli.bess.resume_all()
+        if pause_workers != 'no_pause':
+            cli.bess.resume_all()
 
     if module is None:
         cli.fout.write('  The new module "%s" has been created\n' % ret.name)
 
 
-@cmd('add connection MODULE MODULE [OGATE] [IGATE]',
+@cmd('add connection MODULE MODULE [OGATE] [IGATE] [PAUSE_WORKERS]',
      'Add a connection between two modules')
-def add_connection(cli, m1, m2, ogate, igate):
+def add_connection(cli, m1, m2, ogate, igate, pause_workers):
     if ogate is None:
```

Comment:
Added the default value (it should be "pause" to maintain the same behavior)

Thanks!